File: Core/AATM.Core.UI.Logging/LoggingModule.vb
Public Class LoggingModule
    Implements IAppModule

    Public Sub RegisterServices(container As SimpleDIContainer) Implements IAppModule.RegisterServices
        ' This is where we register the concrete log viewer form with the DI container.
        container.Register(GetType(FrmLogViewer), New FrmLogViewer())
    End Sub

End Class

File: Core/AATM.Core.UI.Logging/LoggingModule.cs

namespace AATM.Core.UI.Logging
{
    public class LoggingModule : IAppModule
    {

        public void RegisterServices(SimpleDIContainer container)
        {
            // This is where we register the concrete log viewer form with the DI container.
            container.Register(typeof(FrmLogViewer), new FrmLogViewer());
        }

    }
}